services/datamanager/builtin/builtin.go

```diff
@@ -532,6 +556,15 @@ func (svc *builtIn) Reconfigure(
 			svc.closeSyncer()
 		}
 	}
+
+	if !svc.captureDisabled {
```

Comment:
Adding here since I want to use the syncer after its been intialized, will have nil guards in the method that actually uses this but I wanted to get all the plumbing in now

Reply:
Why do we need this to be after syncer has been initialized?

Reply:
The file system deletion stuff will require a syncer to do all the in progress path stuff so I create the thread after its been initialized.

Reply:
Doesn't syncer always get set to nil though if sync is disabled?

Reply:
Yes, my plan was to check for nil in the filesystem routine before trying to access it.

Reply:
Ah okay, perfect. That's what the nil guards means :)

Reply:
Lol might have overjargoned the initial sentence oops